test/plugin/scenarios/sofarpc-scenario/config/expectedData.yaml

```diff
@@ -17,6 +17,23 @@ segmentItems:
 - serviceName: sofarpc-scenario
   segmentSize: gt 2
   segments:
+  - segmentId: not null
+    spans:
+      - operationName: HEAD:/sofarpc-scenario/case/healthCheck
```

Comment:
`/healthCheck` should not be included in the file.

Reply:
Removed it.